order.js

```diff
@@ -0,0 +1,90 @@
+
+orderList = [
```

Comment:
It would be better to store orders as an object rather than as an array so we can look up orders easily using ids